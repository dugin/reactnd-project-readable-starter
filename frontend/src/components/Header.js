import React from 'react';
import AppBar from 'material-ui/AppBar';
import Toolbar from 'material-ui/Toolbar';
import Typography from 'material-ui/Typography';
import styled from 'styled-components';
import SelectMenu from "./SelectMenu";

const StyledTypography = styled(Typography)`
   font-weight: 300;
`;

const StyledToolbar = styled(Toolbar)`
   justify-content: space-between;
   display: flex;
`;

const Header = props => {

    return (
        <AppBar  position="static">
            <StyledToolbar >
                <StyledTypography type="title" color="inherit">
                    My Readable
                </StyledTypography>

                <SelectMenu defaultValue='Sort By' options={['vote score', 'created at']}/>
            </StyledToolbar>
        </AppBar>
    );
};

export default Header;