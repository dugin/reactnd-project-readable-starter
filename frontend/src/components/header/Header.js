import React, {Component} from 'react';

class Header extends Component {

    render() {
        return (
            <nav className="navbar navbar-light bg-primary Header">
                <div className="container">
                <span className="h1 navbar-brand ">My Readable</span>
                </div>
            </nav>
        )
    }
}

export default Header;