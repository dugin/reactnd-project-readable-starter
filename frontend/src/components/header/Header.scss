.Header{
  box-shadow: 5px 5px 10px 0 rgba(46, 61, 73, 0.15);

}