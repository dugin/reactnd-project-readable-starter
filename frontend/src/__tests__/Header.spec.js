import React from 'react';
import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme';
import Header from "../components/header/Header";

describe('Header Component', () => {

    const header = shallow(<Header/>);

    it('renders properly', () => {

        expect(toJson(header)).toMatchSnapshot();

    });
});