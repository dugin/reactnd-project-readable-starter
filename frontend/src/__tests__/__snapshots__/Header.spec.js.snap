// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders properly 1`] = `
<div
  className="App"
>
  <Header />
  <main
    className="App__main container"
  />
  <Footer />
</div>
`;
